### PR TITLE
Add accesslint documentation

### PR DESCRIPTION
**Pull request by gemfarmer**
This PR adds documentation to the accessibility scanning README, based on the recent, and successful addition of [AccessLintCI](https://github.com/accesslint/accesslint-ci/) to the 18f site repo with [this PR](https://github.com/18F/18f.gsa.gov/pull/2134).

cc @maya

### CODE REVIEW

File: accessibility_scanning/README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
+## Accessibility Scanning using AccessLint
+
+[AccessLintCI](https://github.com/accesslint/accesslint-ci) is an automated accessibility scanning tool. It configured with CircleCI to leave comments on Pull Requests stating potential accessibility problems with the committed code.
```
Comment (jessieay) [inline accessibility_scanning/README.md]: small typo: `It configured` -> `It is configured`

File: accessibility_scanning/README.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+
+### Set up with Jekyll
+
+The set up is detailed [here](https://github.com/accesslint/accesslint-ci#installation) for Rails, but can also easily be configured with Jekyll sites as follows. In the `circle.yml` file of your repo add the following:
```
Comment (jessieay) [inline accessibility_scanning/README.md]: I think `setup` is one word?

Comment (jessieay) [inline accessibility_scanning/README.md]: Some 18F projects use Travis, do we want to add a config for them?

Comment (gemfarmer) [inline accessibility_scanning/README.md]: I believe it is 😆 

AccessLint doesn't currently support Travis, but I think it is in their roadmap.

Comment (jessieay) [inline accessibility_scanning/README.md]: that's good info! maybe we can add this info to the README?

**Comment by gemfarmer** [comment]
@jessieay made your suggested grammar changes!

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**